patch_api/api/projects_api.py

```diff
@@ -79,6 +79,7 @@ def retrieve_project(self, id={}, **kwargs):  # noqa: E501
 
         :param async_req bool: execute request asynchronously
```

Comment:
Why is the type last on this particular line? 😄

Reply:
Good question. This is all auto-generated and not being a python expert i'm actually not quite sure what this does.